### ISSUE TITLE
Akita CLI generates wrong model filename

### ISSUE THREAD

**Issue by xuxicheta** [datorama/akita#229]
Have generated entity store for angular schema. WIth name "items"
Got
```
export * from './items.query';
export * from './items.store';
export * from './items.service';
export * from './item.model';
```
But file with model named `item.ts`

**Comment by NetanelBasal** [comment]
This is by design. Model is singular.

**Comment by xuxicheta** [comment]
I mean in index.ts this file is `item.model` but in reality its name is `item.ts`
Filename error

**Comment by NetanelBasal** [comment]
Oh, sorry for that. Let me check.

**Comment by NetanelBasal** [comment]
It is fixed in v4.0.4, thanks for the report.
